src/openssl.c

```diff
@@ -11975,6 +11975,47 @@ static int cipher_final(lua_State *L) {
 } /* cipher_final() */
 
 
+static int cipher_get_tag(lua_State *L) {
+	EVP_CIPHER_CTX *ctx = checksimple(L, 1, CIPHER_CLASS);
+	luaL_Buffer tag;
+	size_t tag_size = luaL_checkinteger(L, 2);
+
+	luaL_buffinit(L, &tag);
+
+	if(1 != EVP_CIPHER_CTX_ctrl(ctx, EVP_CTRL_GCM_GET_TAG, tag_size, (void*)luaL_prepbuffsize(&tag, tag_size))) {
+		goto sslerr;
+	}
+
+	luaL_pushresultsize(&tag, tag_size);
+
+	return 1;
+
+sslerr:
+	lua_pushnil(L);
+	auxL_pusherror(L, auxL_EOPENSSL, NULL);
+
+	return 2;
+} /* cipher_get_tag() */
+
```

Comment:
two spaces between functions for consistency

Reply:
Fixed in https://github.com/wahern/luaossl/pull/201/commits/c77a7cf94c96540aa5d38e786fd2166c89bee18e